### DISCUSSION TITLE
"@angular-devkit/build-angular:dev-server 14.1.2 not reloading node_modules

### DISCUSSION THREAD

**Discussion by pelord** [webpack/webpack#16296] [Q&A]
I'm using @angular-devkit/build-angular:dev-server 14.1.2 . I was told, this is webpack running behind.

Here my setup:
I'm developping a lib, (igo2-lib) and a related sub project (igo2) using the lib components.

In a dev mode, I edit my libs, I build and copy the dist folder from the lib to the igo2/node_modules/@igo2 directory. 

Previously, my changes were applied to the started instance. Webpack was detecting the changes. 

Do you know to allow webpack to reload with the new node_modules applied?

**Comment by alexander-akait** [comment]
I think it would be better to ask in `@angular-devkit/build-angular` repo

**Comment by pelord** [comment]
https://github.com/angular/angular-cli/issues/23960
